tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -80,7 +80,8 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
     const void* ptr3 = (void*)((long)ptr1 + 1);
 
     testEqualFunction(functions, ptr1, ptr2, ptr3);
-    testHashFunction(functions, ptr1, ptr2, ptr3);
+    // Disabling hash function check here as performing hash on opaque values cannot be accuratly matched with the reference platform.
+    // testHashFunction(functions, ptr1, ptr2, ptr3);
```

Comment:
Hmm, I think there's value in knowing that our impl for this hash function doesn't collide. The scenario you describe where the first three bits get wiped out is _possible_ but shouldn't really ever happen with the way allocation typically works.

Does it make sense for us to change our hash to a right shift instead and leave this commented out (as it'll be much more prone to failure)?

---
In reply to: [73764075](https://github.com/Microsoft/WinObjC/pull/833#discussion_r73764075) [](ancestors = 73764075,73762989)